src/utils/utils.js

```diff
@@ -589,6 +600,16 @@ export function isInPre(node) {
             getComputedStyle(element).getPropertyValue('white-space') === 'pre')
     );
 }
+/**
+ * Returns true if the given node is or in a Node with ContentEditable false.
+ *
+ * @param {Node} node
+ * @returns {boolean}
+ */
+export function isContentUneditable(node) {
```

Comment:
I would name this `isContentNoneditable`.

Reply:
Well, I would name this `isContentEditable` and invert the computation.

Reply:
But then my conditions are harder to write :cry: 

:+1: